EmbyServer.Test/ApplicationPathHelperTests.cs

```diff
@@ -0,0 +1,83 @@
+using System.Runtime.InteropServices;
+using MediaBrowser.Server.Mono;
+using Moq;
+using NUnit.Framework;
+
+namespace EmbyServer.Test
```

Comment:
Can we not introduce new `Emby`-named namespaces? I think new development should go under `Jellyfin`.

Reply:
I agree, not sure how best to handle this though. Emby's namespaces are also a mess between MediaBrowser and Emby, and often don't match the project name correctly, either.

Reply:
Let's just start by developing new code under `Jellyfin`, in this case I propose to use
```C#
namespace Jellyfin.Test
```
And replace namespaces in other places later as we go.

Reply:
Do we want one namespace for all test projects, or in this case should it be `namespace JellyfinServer.Test`

In the case of one namespace per test project, my suggestion would be 1 namespace per test-project, change this namespace to be `JellyfinServer.Test`, leave the project name EmbyServer.Test, with the aim of renaming EmbyServer and EmbyServer.Test to JellyfinServer(.Test) in a separate PR.

I'd be keen on namespaces accurately describing where the code can be found, and what it relates to, rather than having one large general Test namespace.

Reply:
I like the idea of separate test namespace per project.